code/src/index.css

```diff
@@ -1,13 +1,194 @@
+  -moz-box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3),
+    0 0 40px rgba(0, 0, 0, 0.1) inset;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3), 0 0 40px rgba(0, 0, 0, 0.1) inset;
+}
+
+.todo-information-container {
+  margin: 20px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: #183059;
+}
+
+.main-title {
+  font-size: 26px;
+  text-shadow: 0 1px 4px rgba(0, 0, 0, 0.3);
+}
+
+.task-counter {
+  text-align: right;
+  width: 60px;
+  font-size: 12px;
+  text-shadow: 0 1px 1px rgba(0, 0, 0, 0.2);
+}
+
+.input-field {
+  border-radius: 5px;
+  padding-left: 5px;
+  border: none;
+  resize: none;
+  outline: none;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3);
+}
+
+.todo-form {
+  margin: 20px;
+  display: flex;
+  flex-direction: column;
+}
+
+.add-todo-button {
+  border: 2px solid #305dc9;
+  border-radius: 5px;
+  background-color: #305dc9;
+  transition-duration: 0.4s;
+  color: white;
+  margin-top: 12px;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3);
+}
+
+.add-todo-button:hover {
+  border: 2px solid #183059;
+  background-color: #183059;
+  color: white;
+  cursor: pointer;
+}
+
+.no-todos {
+  margin: 20px;
+  border-radius: 5px;
+  padding: 10px 5px;
+  background-color: white;
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande',
+    'Lucida Sans', Arial, sans-serif;
+  font-size: 12px;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3);
+}
+
+.todo-list {
+  margin: 20px;
+}
+
+.todo-item {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 5px;
+  position: relative;
+  border-radius: 5px;
+  margin-bottom: 10px;
+  background-color: white;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.3);
+  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande',
+    'Lucida Sans', Arial, sans-serif;
+  font-size: 12px;
+}
+
+.checkbox {
+  cursor: pointer;
+  height: 15px;
+  position: absolute;
+  width: 15px;
+}
+
+.todo-text {
+  margin-left: 30px;
+  word-break: break-word;
+}
+
+.delete-button {
+  border: none;
+  background-color: white;
+  color: #183059;
+  font-weight: bolder;
+  font-size: 12px;
+}
+
+.delete-button:hover {
+  cursor: pointer;
+}
+
+@media (min-width: 767px) {
+  .container {
+    width: 50%;
+  }
+
+  .main-title {
+    font-size: 30px;
+  }
+
+  .task-counter {
+    font-size: 16px;
+    width: 80px;
+  }
+
+  .checkbox {
+    height: 18px;
+    width: 18px;
+  }
+
+  .add-todo-button {
+    font-size: 16px;
+    padding: 5px;
+  }
+
+  .input-field,
+  .no-todos,
+  .todo-list,
+  .todo-item,
+  .delete-button {
+    font-size: 16px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .container {
+    width: 40%;
+  }
+
+  .main-title {
+    font-size: 40px;
+  }
+
+  .task-counter {
+    font-size: 20px;
+    width: 100px;
+  }
+
+  .checkbox {
+    height: 20px;
+    width: 20px;
+  }
+
+  .add-todo-button,
+  .input-field,
+  .no-todos,
+  .todo-list,
+  .todo-item,
+  .delete-button {
+    font-size: 20px;
+  }
+}
```

Comment:
It would be nice if you try to split your CSS code into separate CSS files for every component.